tests/conftest.py

```diff
@@ -97,14 +97,14 @@ def images(content: Sphinx) -> List[Path]:
 
 @pytest.fixture()
 def legacy_tex_images(content: Sphinx) -> List[Path]:
-    tex = (content.outdir / "python.tex").text()
+    tex = (content.outdir / "python.tex").read_text()
```

Comment:
This should also be updated with the draw.io update.

Reply:
Done :heavy_check_mark: